files/routable-files/index.js

```diff
@@ -0,0 +1,9 @@
+/*jshint node:true*/
+'use strict';
+
+const EngineAddon = require('ember-engines/lib/engine-addon');
+
+module.exports = EngineAddon.extend({
+  name: '<%= dasherizedModuleName %>'<% if (hasLazyFlag) { %>,
+  lazyLoading: <%= isLazy %><% } %>
```

Comment:
Should this be updated to use the new object definition for `lazyLoading`? Same for the routeless engine blueprint as well.